hashicorp/web-unified-docs
# chore: Remove Vagrant content

# Changes

- Remove vagrant content from code base to prep for education engineer test. This includes:
  - Removing vagrant from the `content` dir
  - Removing vagrant from the `public/assets` dir
  - Commenting out vagrant in `scripts/migrate-content/repo-config.mjs`

- [Asana task](https://app.asana.com/0/1207899860738460/1208561647881344/f)

# Testing

1. Make a curl request or use Postman to send a `GET` request 
```
curl https://web-presence-experimental-docs.vercel.app/api/content/vagrant/nav-data/latest/docs
```
1. You should receive a valid response, such as this:
<img width="1033" alt="Screenshot 2024-10-16 at 16 14 26" src="https://github.com/user-attachments/assets/1285e566-a6ef-4067-aac6-7ee45c652bc2">


1. Make a curl request or use Postman to send a `GET` request
```
curl https://web-presence-experimental-docs-git-heat-chorer-b5d404-hashicorp.vercel.app/api/content/vagrant/nav-data/latest/docs
```
1. You should receive a response of `Not Found`
